homeassistant/components/velbus/diagnostics.py

```diff
@@ -0,0 +1,15 @@
+"""Diagnostics support for Velbus."""
+from __future__ import annotations
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+
+
+async def async_get_config_entry_diagnostics(
+    hass: HomeAssistant, entry: ConfigEntry
+) -> dict:
```

Comment:
Could you make the return value more specific?
```suggestion
) -> dict[str, Any]:
```